File: API/Entities/Address.cs
namespace API.Entities
{
    public class Address
    {
        public int Id { get; set; }
        public string AddressName { get; set; }
        public string Name { get; set; }
        public string Mobile { get; set; }
        public string House { get; set; }
        public string Landmark { get; set; }
        public int LocationId { get; set; }
        public Location Location { get; set; }
        public string PostalCode { get; set; }
        public User User { get; set; }
        public Store Store { get; set; }
    }
}

File: client/src/app/modal/address.ts
export class Address {
  addressName: string;
  house: string;
  landmark: string;
  postalCode: string;
  areaId: number;
  cityId: number;
  stateId: number;
  countryId: number;
  locations: {
    areas: LocationInfo[];
    cities: LocationInfo[];
    states: LocationInfo[];
  };
}

export class LocationInfo {
  id: number;
  name: string;
  parentName: string;
}
